### PR TITLE
UMETT Guestblog

### PR DESCRIPTION
**Pull request by jpechmann**
Pull request description
------------------------

### CODE REVIEW

File: _posts/2017-03-30-latest-news-from-utah-s-mapping-emergency-technical-team.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+In order to ensure our response team members are well prepared to work in an emergency response environment, volunteers must complete some training requirements.
+
+There will be a pre-conference training at the UGIC Conference in Park City on May 9th to introduce the Incident Command System, the basic command structure for emergency response, as well as details on how the UMETT response group will operate. The instructor-led course will cover ICS-100, 200, 700, and 800 in a single session and you will receive official FEMA completion certificates. For those of you working in government agencies, this training may already be required for you and this is an excellent opportunity to complete them all in a single day. You can register for the pre-conference training and the rest of the conference on [UGIC's Conference webpage](http://ugic.org/ugic/ugic-conference-2017/). 
+
+If you would like to be on the UMETT mailing list for future meetings and other related announcements, please contact Talsan Schulzke at [tschulzke@ufa-slco.org](mailto:tschulzke@ufa-slco.org) or Kate Smith at [katesmith8@gmail.com](katesmith8@gmail.com).
```
Comment (steveoh) [inline _posts/2017-03-30-latest-news-from-utah-s-mapping-emergency-technical-team.md]: do you want the mailto for kate's email also?

Comment (jpechmann) [inline _posts/2017-03-30-latest-news-from-utah-s-mapping-emergency-technical-team.md]: @steveoh oops. yes.

Comment (jpechmann) [inline _posts/2017-03-30-latest-news-from-utah-s-mapping-emergency-technical-team.md]: Fixed!

**Comment by steveoh** [comment]
holding off until you do your image stuff

**Comment by jpechmann** [comment]
@steveoh image added. good to go!

**Pull request merged**